src/App.js

```diff
@@ -0,0 +1,42 @@
+// App.js
+import React from 'react';
+import MyComponent from './components/MyComponent';
+import './App.css';
+
+const App = () => {
+  return (
+    <div class="rectangle-container">
+      <input style={{ display: 'none' }} type="file" accept="image/*" />
+      <br />
```

Comment:
The spacing given by the `br` can be accomplished using display flex and the [gap property](https://css-tricks.com/snippets/css/a-guide-to-flexbox/#aa-gap-row-gap-column-gap); this property can also help you to position all the elements in the container easily.